analyzer/src/funTest/assets/projects/external/godep-expected-output.yml

```diff
@@ -47,7 +47,7 @@ packages:
       hash: ""
       hash_algorithm: ""
     vcs:
-      type: "GoDep"
+      type: "git"
       url: "github.com/kr/fs"
```

Comment:
IIRC the idea was to put "GoDep" there because it understands URLs like this `github.com/kr/fs`, but Git itself doesn't.

Reply:
Ok, but isn't it confusing to use a type here for which we don't have a VCS implementation? I was seeing "github.com/kr/fs" simply as un-normalized VCS URLs, which is something our normalization properly fixes. So, bottom line, would you prefer to keep the code as-is?

Reply:
Ah, right, that's why it was "GoDep". Hard-coding "git" definitely will produce incorrect data in some cases. And the normalization in ORT won't always work either. The only ways are to invoke `go get` on the URL and then examine the directory to which the code was downloaded or reimplement the "VCS discovery algorithm" of `go get`.

Reply:
No, just wanted to mention it. It is indeed confusing.
Would our normalizer be able to fix those URLs? In the implementation the packages are installed using `go dep` and `vcsProcesed` is read from the cloned project.

Reply:
> will produce incorrect data in some cases.

Could you provide an example?

> The only ways are to invoke `go get` on the URL and examine the directory

Hmm, so then it could be that the URL used by `go get` does not even point to a VCS, but to something completely different, like a ZIP archive (that some version of `go get` happens to support / understand and transparently unpacks), right?

Reply:
> Could you provide an example?

Any import URL that points to a non-git repo, like launchpad.net (bzr).

> Hmm, so then it could be that the URL used by go get does not even point to a VCS, but to 
> something completely different, like a ZIP archive (that some version of go get happens to 
> support / understand and transparently unpacks), right?

So far it's only VCS repositories, but they could add downloading source archives to `go get`, yes.